bin/improver-weighted-blending

```diff
@@ -106,7 +110,7 @@ def calculate_blending_weights(cube, blend_coord, method, wts_dict=None,
             weights_dict = json.load(wts)
         weights_cube = ChooseWeightsLinear(
             weighting_coord, weights_dict,
-            config_coord_name=blend_coord).process(cube)
+            config_coord_name=dict_coord).process(cube)
```

Comment:
There is a bit of tidy up work we could raise relating to the weights cubes as well (if it doesn't exist) when we have abundant free time. The cubes produced here have a threshold coordinate, which is currently redundant (until we one day want to weight different thresholds differently??....yuck). They also contain the 'relative_to_threshold' attribute. None of this really matters as they are thrown away, but they could be tidier.

Reply:
I thought that Fiona's work had dealt with this?

Please can we not raise specific follow-on work relating to metadata (eg the "relative_to_threshold" attribute) OR the blending refactor (which will be needed to meet the new metadata principles) until the appropriate epics have been written.  We are at risk of doing this bittily and creating more problems again.

Reply:
I thought I did get rid of the threshold dimension. I thought I changed it so that they only return weights along the dim being collapsed. I didn't think about attributes though so that will need to be fixed at some point if it is being inherited.

Reply:
I've created https://github.com/metoppv/improver/issues/805 to cover this.